wire-ios-sync-engine/Source/Data Model/Conversation+Join.swift

```diff
@@ -19,11 +19,12 @@
 import Foundation
 
 public enum ConversationJoinError: Error {
-    case unknown, tooManyMembers, invalidCode, noConversation, guestLinksDisabled
+    case unknown, tooManyMembers, invalidCode, noConversation, guestLinksDisabled, invalidConversationPassword
```

Comment:
What do you think about "invalidGuestLinkPassword" ? Will this be more readable?

Reply:
The reason I added `invalidConversationPassword` is because that's the error label provided in swagger. I wanted both to match in terms of name